includes/code-analysis/suppress-warning.md

```diff
@@ -0,0 +1,11 @@
+## Suppress a warning
+
+To suppress a rule violation, set the severity option for the specific rule ID to `none` in an EditorConfig file. For example:
+
+```ini
+dotnet_diagnostic.CA1822.severity = none
+```
```

Comment:
I had a mistake a couple of times before when creating a new editorconfig, which is missing `[*.cs]` or `[*.{cs,vb}]`, etc above the configuration. The configurations doesn't seem to have any effect without this. It would be good if this was explained.